src/utils/computeFluxMod.F90

```diff
@@ -5,24 +5,21 @@ module computeFluxMod
   use shr_kind_mod      , only : r8 => shr_kind_r8
   implicit none
 
+  type, abstract :: flux_type
```

Comment:
**Comment from @martynpclark Sep 8, 2017**

This is pedantic, I know, though can we call this `computeFlux_type` (or something similar)? The reason is that it is easy to confuse `flux_type` with `WaterFlux_type` and `EnergyFlux_type`, and folks could assume that the type only includes fluxes (whereas, in fact, it will include all variables required to compute fluxes).

Reply:
**Comment from @billsacks Sep 11, 2017**

Fixed - thanks. Not pedantic at all - this is a very good point.